dom0-updates/qubes-dom0-update

```diff
@@ -55,9 +55,6 @@ QVMRUN_OPTS=
 CLEAN=
 TEMPLATE=
 TEMPLATE_BACKUP=
-FW_UPDATE=
```

Comment:
Are those changes simply reverting your previous changes? I would prefer to create a clean branch targeting the original code. It would be easier to upstream later.